app/assets/stylesheets/base/color_and_text_styles.scss

```diff
@@ -0,0 +1,344 @@
+/* Color palette */
+:root {
+  --theme-primary: #f3c937;
+  --theme-secondary: #333333;
+  --text-title: #000000;
+  --text-subtitle: #aaaaaa;
+  --text-body: #555555;
+  --element-toned-down: #bbbbbb;
+  --text-toned-down: #777777;
+  --text-light: #ffffff;
+  --n7755-cc: #7755cc;
+  --facebook-blue: #4267b2;
+  --error: #ee4444;
+  --tealish: #22aa99;
+  --rosa: #ff88aa;
+  --light-moss-green: #99cc66;
+  --n1199-ee: #1199ee;
+  --light-hover: rgba(0, 0, 0, 0.1);
+}
+
+/* Text styles */
+title {
+  font-family: Roboto;
+  font-size: 24px;
+  font-weight: 900;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.33;
+  letter-spacing: 0.2px;
+  color: var(--text-title);
```

Comment:
i don't think we should be using css variables for browser support, can you change these to sass variables?

Reply:
I haven't really looked in browser support between both of them. These css variables were just copied from what was on zeplin, so I didn't really bother. But sure, I can change it.